NEWS.md

```diff
@@ -76,6 +76,11 @@
 * Added `stat_qq_line()` to make it easy to add a simple line to a Q-Q plot. This
   line makes it easier to judge the fit of the theoretical distribution 
   (@nicksolomon).
+  
+* Added `tag` label for adding identification tags to the plot. A tag is added 
```

Comment:
I think you need a brief explanation of what you might use a tag for here

Reply:
Fixed in 5f70fd8